README.md

```diff
@@ -57,13 +133,14 @@ If you want to perform this data collation inside pycytominer using the `cyto_ut
 ```bash
 # Example for general case commit:
 pip install "pycytominer[collate] @ git+git://github.com/cytomining/pycytominer"
+
 # Example for specific commit:
 pip install "pycytominer[collate] @ git+git://github.com/cytomining/pycytominer@2aa8638d7e505ab510f1d5282098dd59bb2cb470"
 ```
 
 If using `pycytominer` in a conda environment, in order to run `collate.py`, you will also want to make sure to add `cytominer-database=0.3.4` to your list of dependencies.
 
-## Creating a cell locations lookup table
+### Creating a cell locations lookup table
 
 The `CellLocation` class offers a convenient way to augment a [LoadData](https://cellprofiler-manual.s3.amazonaws.com/CPmanual/LoadData.html) file with X,Y locations of cells in each image.
 The locations information is obtained from a single cell SQLite file.
```

Comment:
Install instructions in line 152 should also be changed to match above approach.